public-api/src/main/java/org/pmiops/workbench/publicapi/DataBrowserController.java

```diff
@@ -714,58 +749,22 @@ public SurveyQuestionResult apply(AchillesResult o) {
             }
         }
 
+        if (conceptList.size() == 0) {
+            throw new DataNotFoundException("There are no concepts that match these criteria to fetch");
+        }
```

Comment:
Not sure you need to throw an exception here. It seems perfectly valid for a concept search to return an empty list

Reply:
Thanks.